governance/program/src/state/enums.rs

```diff
@@ -155,12 +155,17 @@ pub enum VoteThreshold {
     /// The minimum number of votes in % out of the entire pool of governance tokens eligible to vote
     /// which must be cast for the vote to be valid
     /// Once the quorum is achieved a simple majority (50%+1) of Yes votes is required for the vote to succeed
-    /// Note: Quorum is not implemented in the current version
+    // TODO: if quorum is achieved how it's considered about veto and deny votes?
     QuorumPercentage(u8),
```

Comment:
Oh, `AttendanceQuorum(5000, 0)` does not work as `all succeeded` because of `deny_weigth` https://github.com/solana-labs/solana-program-library/blob/master/governance/program/src/state/proposal.rs#L403